app/backend/lib/sso-middleware.ts

```diff
@@ -23,6 +26,10 @@ export default async function ssoMiddleware() {
     getLandingRoute: () => {
       return '/dashboard';
     },
+    bypassAuthentication: {
+      login: mockAuth,
+      sessionIdleRemainingTime: mockAuth,
```

Comment:
Do we want to turn both options on automatically when mockAuth = true? Is there ever a case where we want `sessionIdleRemainingTime` to be `false` while `login` is `true`? From looking at the SSO repo, it seems that the check to the sessionIdleRemainingTime would return a `0`, so it would be useful to have some testing for when a user's session expires. 

Not needed for this PR I think, but something to think about nonetheless.